services/src/api/model/services.rs

```diff
@@ -0,0 +1,25 @@
+use crate::api::model::operators::{
+    GdalMetaDataList, GdalMetaDataRegular, GdalMetaDataStatic, GdalMetadataNetCdfCf, MockMetaData,
```

Comment:
You have to duplicate these types here as well and implement `ToSchema` for them